### ISSUE TITLE
OpenGL ES 2.0 backend

### ISSUE THREAD

**Issue by ShFil119** [rwengine/openrw#418]
I think we should start think about it, later or earlier.

**Comment by JayFoxRox** [comment]
Last time I worked on OpenRW we were discussing integration or librw by aap. This would severly impact design decisions for renderers.

I think it's too early to think about OpenGL ES anyway. Especially becaue we don't know yet which other APIs might become popular before OpenRW is even usable (which will be a couple of years probably).

We should focus on gameplay aspects for now.

*(Topic can be closed in my opinion. I'll leave it opened for now as others might disagree)*

**Comment by darkf** [comment]
Agreed that it's fruitless at this point in time, until the game actually works to a reasonable degree. It's something to consider in the future, however.

**Comment by Askmewho** [comment]
mmmm, and please mention the need of opengl 3.3 on the main page. it doesnt work with my opengl 2.1 setup obviously...
